assets/armsnippets.jsonc

```diff
@@ -1884,19 +1888,20 @@
             "{",
             "    \"name\": \"[toLower('${1:ubuntuVM1}storage')]\",",
             "    \"type\": \"Microsoft.Storage/storageAccounts\",",
-            "    \"apiVersion\": \"2015-06-15\",",
+            "    \"apiVersion\": \"2019-06-01\",",
             "    \"location\": \"[resourceGroup().location]\",",
             "    \"tags\": {",
             "        \"displayName\": \"${1:ubuntuVM1} Storage Account\"",
             "    },",
-            "    \"properties\": {",
-            "        \"accountType\": \"Standard_LRS\"",
-            "    }",
+            "    \"sku\": {",
+            "       \"name\": \"${2:Standard_LRS}\"",
+            "    },",
+            "    \"kind\": \"${3|Storage,StorageV2,BlobStorage,BlockBlobStorage,FileStorage|}\"",

```

Comment:
I would think not all of these would be appropriate for use as a storage account for a VM.  My guess would be StorageV2 would be recommended, but you should see if there's a standard one recommended.

Reply:
Yes, good point. Looking into this now.

Reply:
Fixed / aligned with portal